Buildings/HeatTransfer/Data/OpaqueConstructions.mo

```diff
@@ -11,7 +11,8 @@ package OpaqueConstructions
       annotation (choicesAllMatching=true, Evaluate=false, Placement(transformation(extent={{60,60},{80,80}})));
    final parameter Real R(unit="m2.K/W")=sum(material[i].R for i in 1:nLay)
       "Thermal resistance per unit area";
-
+   parameter Integer nSta2[nLay](each min=1)={material[i].nSta for i in 1:nLay}
```

Comment:
This shouldn't be set to `final` as they will be modified in `Buildings.Fluid.HeatExchangers.RadiantSlabs.SingleCircuitSlab`.